hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/DefaultSource.scala

```diff
@@ -177,6 +192,70 @@ class DefaultSource extends RelationProvider
 
   override def shortName(): String = "hudi_v1"
 
+  private def getBaseFileOnlyView(useHoodieFileIndex: Boolean,
```

Comment:
these codes have been discarded. please adapt the new `BaseFileOnlyViewRelation`.

Reply:
Yes, this is a issue I want to discuss with you, BaseFileOnlyViewRelation cannot support vector read/ DPP,   We ran over 1t of tpcds, and the performance decreased significantly
I will use another PR to support BaseFileOnlyViewRelation.

Reply:
@xiarixiaoyao : Are we regressing to poor performance with this change ? @YannByron mentions that this code has been discarded.

Reply:
@bvaradar  yes i will try to paste a simple perf result， tomorrow. 
Adapting this change is very easy, but I don't want to lose too much performance